api/policies/isInGame.js

```diff
@@ -7,6 +7,8 @@
  * @docs        :: http://sailsjs.org/#!/documentation/concepts/Policies
  *
  */
+
+// TODO #965 - Delete this policy (no longer using req.session.game)
```

Comment:
I hope to remove all use of `req.session.game` and replace it with `req.params.game` so that "which game we are talking about" is always specified via url path. This way we remove the limitation that a user can only request games based on what their session says they're currently in (only one game) and instead track authorization to access a given game purely by database queries i.e. `game.p0` and `game.p1` and use the url to determine which game you're looking at.

This should let us start allowing users to have multiple tabs open and play multiple games simultaneously